package.json

```diff
@@ -52,6 +52,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
+    "@types/semver": "^7.3.4",
```

Comment:
I'm a little surprised that `semver` is working because the line above is only supposed to add the TypeScript definitions of semver.  I think you might be getting lucky because some other library pulled in `semver`.  But here is what it should be:
* `@types/semver` in the _dev_ dependencies
* `semver` in the dependencies

To fix this:
```
npm remove @types/semver --save
npm add semver --save
npm add @types/semver --save-dev
```

Reply:
Ahhh yes, I actually think this still works because I initially did import `server` and ran an `npm install`, then got confused and thought I'd need `@types/semver` haha. I'll revert this back in the next commit!